sa/sa.go

```diff
@@ -2143,3 +2148,67 @@ func (ssa *SQLStorageAuthority) IncidentsForSerial(ctx context.Context, req *sap
 	}
 	return incidentsForSerial, nil
 }
+
+// SerialsForIncident queries the provided incident table and returns the
+// resulting rows as a stream of `*sapb.IncidentSerial`s. The caller receives
+// this stream of serials by iterating over the `Recv()` method of the returned
+// streaming client. An `io.EOF` error signals that there are no more serials to
```

Comment:
This is the server-side implementation. It shouldn't document how the client processes the response; that's part of the gRPC API. Specifically let's delete the sentence "The caller receives ... streaming client."